ui/imports/shared/controls/TransactionDelegate.qml

```diff
@@ -487,7 +483,12 @@ StatusListItem {
         return details.replace(/[\r\n\s]*$/, '')
     }
 
-    function getSubtitle(allAccounts) {
+    function getSubtitle(allAccounts, description) {
+        if (root.isCommunityAssetViaAirdrop) {
+            const image = description ? "" : " <img src='" + root.communityImage + "' width='18' height='18' </img>"
+            return qsTr("%1 (community asset) from%2 %3 via %4").arg(root.transactionValue).arg(image).arg(root.communityName).arg(root.networkName)
```

Comment:
```suggestion
            return qsTr("%1 (community asset) from %2 %3 via %4").arg(root.transactionValue).arg(image).arg(root.communityName).arg(root.networkName)
```

Reply:
I get it as a developer :) However, as a translator, I would have assumed that the dev made a typo there and I'd just insert the missing space. Can you just remove the space from the `img` tag?